unlock-app/src/components/interface/locks/Settings/forms/CreditCardForm.tsx

```diff
@@ -67,7 +67,7 @@ const DisconnectStripe = ({
       <div className="flex flex-col items-center gap-4 md:gap-8 md:flex-row">
         <Badge variant="green" className="justify-center w-full md:w-1/3">
           <div className="flex items-center gap-2">
-            <span>Payment method enabled</span>
+            <span>Fiat payments enabled</span>
```

Comment:
Maybe the wording can be simpler here? Most people arent familiar with "fiat" as a concept...